mass_sorting/views/mass_sort_wizard_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2016-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<openerp><data>
+
+    <record id="view_mass_sort_wizard_form" model="ir.ui.view">
+        <field name="model">mass.sort.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Mass Sorting" version="7.0">
+                <group col="4">
+                    <field name="description" nolabel="1" colspan="4" attrs="{'invisible': [('allow_custom_setting', '=', True)]}"/>
+                    <field name="line_ids" colspan="4" nolabel="1" attrs="{'invisible': [('allow_custom_setting', '=', False)]}">
+                        <tree editable="bottom">
+                            <field name="sequence" widget="handle"/>
+                            <field name="field_id"/>
```

Comment:
Ow ! I don't understand why change a widget impact the domain !

Well, I tried, 

    domain="[('model', '=', one2many_model)]"
-> NameError: name 'one2many_model' is not defined

    domain="[('model', '=', parent.one2many_model)]"
-> NameError: name 'parent' is not defined

(and other like     domain="[('model', '=', 'parent.one2many_model')]" with quotes but no items are displayed).

Do you have an idea how to do that ?